lib/matplotlib/backends/backend_mixed.py

```diff
@@ -139,8 +139,8 @@ def stop_rasterizing(self):
                 #       backends support this.
                 self._renderer.draw_image(
                     gc,
-                    float(l) / self.dpi * self._figdpi,
-                    (float(height)-b-h) / self.dpi * self._figdpi,
+                    l / self.dpi * self._figdpi,
+                    (height-b-h) / self.dpi * self._figdpi,
```

Comment:
To me, since you are in here, these would be much clearer as
```python
self._figdpi / self.dpi,
self._figdpi * (height-b-h) / self.dpi,
```
Seeing `a/b*c` always slows me down in a way that `c*a/b` or `(a/b)*c` doesn't.

Reply:
agreed, fixed